tests/integration/targets/sysctl/tasks/main.yml

```diff
@@ -217,8 +217,14 @@
 - name: Test on RHEL VMs
   when:
     - ansible_facts.virtualization_type != 'docker'
-    - ansible_facts.distribution == 'RedHat'
+    - ansible_facts.system == 'Linux'
```

Comment:
please try with:
```yaml
    when:
      - ansible_facts.virtualization_type not in ['docker', 'containerd']
```
and no more (other hosts than Linux are skipped from `aliases`)

Reply:
@quidame 
I really appreciate your help! It seems that OSX doesn't support `systemctl -p` option. However, I think the `-p` option is a requirement of the sysctl module.
So I think we probably need to skip OSX as well. I would appreciate it if you could give me your opinion :)
* https://dev.azure.com/ansible/ansible.posix/_build/results?buildId=22948&view=logs&j=a672a2ca-850f-5934-0d20-eacaaa0e01bc&t=79862c18-8b9e-5d82-2fb6-545ffdaaf722&l=867

Reply:
oh yes, my first hypothesis is that (for whatever reason) `skip/macos` is needed in addition to `skip/osx` in `tests/integration/targets/sysctl/aliases` to skip **all** Mac OS X on the CI platform :)

If it doesn't work with this, so you could add a condition such as `ansible_system == 'Linux'` in the related blocks/tasks.

Or even do both, as you feel. This is not so much important at this step. We just know that we target Linux: saying it in `aliases` is less resources-consuming than saying it in the tasks.

Reply:
yup, thanks a lot!
I have added `ansible_system == 'Linux'` :)